### PR TITLE
POM update

### PR DESCRIPTION
**Pull request by jglick**
Routine bump in preparation for `plugin-compat-tester` runs against JEP-200.

@reviewbybees

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -4,26 +4,27 @@
...
 
   <artifactId>cobertura</artifactId>
   <packaging>hpi</packaging>
   <version>1.13-SNAPSHOT</version>
   <name>Jenkins Cobertura Plugin</name>
-  <url>http://wiki.jenkins-ci.org/display/JENKINS/Cobertura+Plugin</url>
+  <url>https://wiki.jenkins.io/display/JENKINS/Cobertura+Plugin</url>
 
   <properties>
     <jenkins.version>2.13</jenkins.version>
```
Comment (oleg-nenashev) [inline pom.xml]: I would bump it to 2.19.3 so that tests run for LTS

Comment (jglick) [reply pom.xml]: Agreed, but leaving that to the plugin maintainer to do later if he agrees.

**Comment by jglick** [comment]
@reviewbybees done

File: pom.xml (hunk 1 above)
Comment (jeffpearce) [reply pom.xml]: Makes sense to me, but why not 2.19.4?

Comment (oleg-nenashev) [reply pom.xml]: 2.19.4 works as well. I just forgot we have .4 in that LTS line

**Pull request merged**